docs/v0.5.x/usage-overview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Usage Overview
+---
+
+Once you've installed your plugins and configured your deployment strategy you're ready to **deploy**!
+
+ember-cli-deploy currently supports 3 commands, each of them takes a first mandatory argument `<deployTarget>` (i.e. `'production'`) and some optional parameters.
+
+### `ember deploy`
+
+Runs the main [deploy pipeline](../pipeline-hooks/#hooks-by-command) that takes care of deploying your assets and your `index.html` file.
+
+It optionally takes a `--activate` flag that also triggers the `activate*` hooks passing the `revision` that just got uploaded.
+
+**Example:**
+
+```
+ember deploy production
+```
+
+### `ember deploy:activate`
+
+Runs the `activate*` hooks that are normally configured by your index plugin and are used to select a `revision` as the primary one to be served by your backend.
```

Comment:
There is no concept of an "index plugin" any more. Perhaps  "Runs a pipeline with the `activate*` hooks that some plugins implement to make a specific `revision` as the active one to be served by your backend."